### ISSUE TITLE
Crash when extracting enchantments from pick in Enchantment Extractor

### ISSUE THREAD

**Issue by calloatti** [Buuz135/Industrial-Foregoing#304]
Industrial Foregoing Version:  industrialforegoing-1.12.2-1.8.10-142.jar
TeslaCoreLib Version: tesla-core-lib-1.12-1.0.13.jar 
Crashlog (If Any):  https://paste.ee/p/k2SN2

This started happening after I updated to the version specified.

**Comment by calloatti** [comment]
Seems like the pick NBT data was somehow corrupt. I managed to remove all the enchantments from the pick now using the Enchantment Extractor, but this is how the NBT looked like:

Left: abnormal pick with all enchants removed, Right: normal pick

![image](https://user-images.githubusercontent.com/12817862/39149959-2c53849a-4717-11e8-9f52-0190b8c98db6.png)

Since I only used IF machines to apply/remove enchantments, I guess something went wrong along the way.

I got into this problem trying to upgrade some enchantments by one level by using Quark's Tomes, so I first removed the enchantments and tthen upgraded some of them and finally tried to apply them back.

After getting a new pick, I managed to do the entire process with no crashes.
